config/routes.rb

```diff
@@ -25,4 +25,11 @@
   resources :categories, only: [:show]
   resources :queue_items, only: [:create, :destroy]
   resources :followerships, only: [:create, :destroy]
+
+  get '/forgot_password', to: 'password_resets#new'
```

Comment:
`get '/forgot_password', to: 'password_resets#new', as: :forgot_password` if you need an example.